module_utils/network_lsr/argument_validator.py

```diff
@@ -1064,6 +1064,13 @@ def _validate_post_state(self, value, name, result):
         """
         actions = []
         state = result.get("state")
+        persistent_state = result.get("persistent_state")
```

Comment:
Looking at the lines following this addition, I think this code should be merged since one condition is repeated here. And `persistent_state_default` does not make sense to have anymore. See my other comment for details.